cuda_pathfinder/tests/child_load_nvidia_dynamic_lib_helper.py

```diff
@@ -24,15 +26,20 @@ def validate_abs_path(abs_path):
 
 
 def child_process_func(libname):
-    from cuda.pathfinder import load_nvidia_dynamic_lib
+    from cuda.pathfinder import DynamicLibNotFoundError, load_nvidia_dynamic_lib
     from cuda.pathfinder._dynamic_libs.load_nvidia_dynamic_lib import _load_lib_no_cache
     from cuda.pathfinder._dynamic_libs.supported_nvidia_libs import (
         IS_WINDOWS,
         SUPPORTED_LINUX_SONAMES,
         SUPPORTED_WINDOWS_DLLS,
     )
 
-    loaded_dl_fresh = load_nvidia_dynamic_lib(libname)
+    try:
+        loaded_dl_fresh = load_nvidia_dynamic_lib(libname)
+    except DynamicLibNotFoundError:
+        sys.stdout.write("CHILD_LOAD_NVIDIA_DYNAMIC_LIB_HELPER_DYNAMIC_LIB_NOT_FOUND_ERROR:\n")
```

Comment:
This seems unnecessarily low-level (`sys.stdout.write` + a newline). Why not use `print`?

```suggestion
        print("CHILD_LOAD_NVIDIA_DYNAMIC_LIB_HELPER_DYNAMIC_LIB_NOT_FOUND_ERROR:")
```